### DISCUSSION TITLE
Redirect Blogger URL to Mecha

### DISCUSSION THREAD

**Discussion by ghost** [mecha-cms/mecha#107] [Q&A] [answered]
Ada ekstensinya, Mas?

**Comment by taufik-nurrohman** [comment, marked as answer]
Kalau mau nulis URL satu-satu bisa pakai ekstensi [_kick_](https://mecha-cms.com/store/extension/kick). Kalau mau otomatis bisa pakai kode PHP. Tapi tergantung pola URL seperti apa.
